javascript/javascript3/week3/ArtUsingClasses/artusing.js

```diff
@@ -0,0 +1,53 @@
+// const canvas = document.getElementById("canvas");
+// const context = canvas.getContext("2d");
+// const centerX = canvas.width / 2;
+// const centerY = canvas.height / 2;
+// const radius = 70;
+
+// context.beginPath();
+// context.arc(centerX, centerY, radius, 0, 2 * Math.PI, false);
+// context.fillStyle = "pink";
+// context.fill();
+// context.lineWidth = 3;
+// context.strokeStyle = "#003300";
+// context.stroke();
```

Comment:
don't include long comment sections in your code. Remove and git saves your history if you ever need it again.